incubator/swift/README.md

```diff
@@ -35,4 +35,34 @@ Templates are used to create your local project and start your development. When
     Hello, world!
     ```
 
-**NOTE:** Currently the `appsody deploy` command only works for deploying web applications.
\ No newline at end of file
+**NOTE:** Currently the `appsody deploy` command only works for deploying web applications.
+
+## Enabling an existing Project
+
+The Swift stack can be used with an existing server-side Swift project in order to provide an interative containerized development and test environment, and to "cloud package" it into an optimized production Docker container.
```

Comment:
interative **->** interactive

Reply:
Fixed the typo, but kept the user of "iterative" rather than switched to "interactive".